django/applications/catmaid/control/label.py

```diff
@@ -222,7 +223,7 @@ def labels_for_nodes(request, project_id=None):
     return JsonResponse(result)
 
 @requires_user_role(UserRole.Annotate)
-def label_update(request, project_id=None, location_id=None, ntype=None):
+def label_update(request:HttpRequest, project_id=None, location_id=None, ntype=None) -> JsonResponse: # XXX ntype would be type str except it has a default of None so it must be Optional[str], which makes get_link_model angry.
```

Comment:
It would be fine to remove the default values for all three parameters. All of them are expected to be passed in for this API.